www/src/css/components/_card-grid.scss

```diff
@@ -0,0 +1,105 @@
+.card-grid {
+  display: grid;
+  grid-column-gap: 15px;
+  grid-row-gap: 15px;
+  grid-auto-flow: dense;
+}
+
+.card-grid-3 {
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+}
+.card-grid-4 {
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+}
+.discord-banner {
```

Comment:
minor suggestion: I’d recommend either componentizing this, or scoping it.

**Componentizing**
Take the reusable part, and make that the base, e.g.:

```scss
.banner {
  // common code

  // modifiers
  &__discord {
  }
}
```

And in HTML, you’d use `<div class="banner banner__discord">`. This will let you add other similar banners in a similar way.

**Scoping**
However, say that this is only used within `.card`s, and nowhere else. In that case, I’d recommend scoping it to `.card` itself, e.g. `.card-banner` or `.card-discord-banner`, etc. That indicates that this isn’t reusable, and should only ever be in this one “card” context.

But leaving this here just makes it hard to find, and its generic name encourages people to extend/modify this, or use it in the wrong context, which may lead to something breaking down the line.